lib/http/HttpClient_Curl.hpp

```diff
@@ -128,6 +128,8 @@ class CurlHttpOperation {
         // TODO: expose SSL cert verification opts via ILogConfiguration
         curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, 0);      // 1L
         curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, 0);      // 2L
+        // HTTP/2 please, fallback to HTTP/1.1 if not supported
+        curl_easy_setopt(curl, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_2_0);
```

Comment:
The new line of API call says using version 2.0, will it fallback automatically, or any extra parameter needed?

Reply:
Yes, it will fallback automatically to using HTTP/1.1 if 
 - curl is built without the nghttp2 library.
 - the collector is not supporting HTTP/2.0